### PR TITLE
Feature/e UI 2897 assign claim task

### PR DESCRIPTION
**Pull request by ficklephil**
No Unit tests as yet.
Flows for 500, 403, 401, 400 errors working.
Flow for success working.

### CODE REVIEW

File: api/test/pact/constants/work-allocation/tasks.spec.ts
Comment (paul-graham) [inline api/test/pact/constants/work-allocation/tasks.spec.ts:91]: I'd probably call this NAT_AUTHORISED but I like the idea of a Jr suffix. :)

File: src/work-allocation/containers/available-tasks/available-tasks.component.ts
Comment (paul-graham) [inline src/work-allocation/containers/available-tasks/available-tasks.component.ts:130]: These don't need to be string literals as there are no variables to inject.

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts
Comment (paul-graham) [inline src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts:4]: I'd guess this will give a lint warning as they're not alphabetical. :)

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts
Comment (paul-graham) [inline src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts:10]: And I think this will be another lint warning as the imports also need to be alphabetical.

File: src/work-allocation/enums/index.ts
Comment (paul-graham) [inline src/work-allocation/enums/index.ts:7]: Lint warning.

File: src/work-allocation/models/comms/infomation-message.model.ts
Comment (paul-graham) [inline src/work-allocation/models/comms/infomation-message.model.ts:6]: Weird indenting.

File: src/work-allocation/services/work-allocation-task.service.ts
Comment (paul-graham) [inline src/work-allocation/services/work-allocation-task.service.ts:55]: SHould I be mentioning this? Probably not. ;)

File: src/work-allocation/models/comms/infomation-message.model.ts
Comment (ficklephil) [reply src/work-allocation/models/comms/infomation-message.model.ts:6]: Good spot.

File: api/test/pact/constants/work-allocation/tasks.spec.ts
Comment (ficklephil) [reply api/test/pact/constants/work-allocation/tasks.spec.ts]: Done

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts
Comment (ficklephil) [reply src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts]: Ah yep. Done.

Comment (ficklephil) [reply src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts]: Good spot!

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts
Comment (ficklephil) [reply src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts]: Done.

**Review by paul-graham — APPROVED** [review]
Looks fine to me. 👍

**Review by connorpgpmcelroy — APPROVED** [review]
Going to have to do a merge with this branch because this will affect my code

**Comment by paul-graham** [comment]
> Going to have to do a merge with this branch because this will affect my code

Might be better to wait until it's been merged into `workallocation-3`. :)

**Pull request merged**
